src/protocols/ssh/ssh.c

```diff
@@ -191,6 +192,10 @@ void* ssh_client_thread(void* data) {
         return NULL;
     }
 
+    /* Initialize a ttymode array */
```

Comment:
Point of clarification - nothing is being initialized here, only declared. With that much being obvious (that the array is being declared), rather than change "Initialize" to "Declare", I'd say just kill this line.